pilot/pkg/features/security.go

```diff
@@ -88,6 +88,11 @@ var (
 		return res
 	}()
 
+	EnableExternalNodeAuthorizer = env.Register(
```

Comment:
I don't think we need this, ambient is already alpha so we don't need feature flags for additions to it IMO

Reply:
Thank you for your suggestion. In fact, this implementation of multi-cluster node authorizer could also work with one primary cluster. Following that, we will remove this feature flag and enable the multi-cluster node authorizer by default.